src/components/header.tsx

```diff
@@ -1,11 +1,39 @@
 import { st, classes } from './header.st.css';
 import stylableLogo from '../assets/stylable.svg';
+import { useEffect, useState } from 'react';
 
 export interface HeaderProps {
     className?: string;
 }
 
 export const Header: React.FC<HeaderProps> = ({ className }) => {
+    const [isCopied, setIsCopied] = useState(false)
+
+    const onShareClick = async () => {
+        const url = window.location.href;
+
+        if ('share' in navigator) {
```

Comment:
this copy to clipboard logic is not react related can be moved outside of the component.